rollbar.go

```diff
@@ -231,8 +231,8 @@ func SetCheckIgnore(checkIgnore func(string) bool) {
 // SetPerson information for identifying a user associated with
 // any subsequent errors or messages. Only id is required to be
 // non-empty.
-func SetPerson(id, username, email string) {
-	std.SetPerson(id, username, email)
+func SetPerson(id, username, email string, opts personOption) {
```

Comment:
Should this be variadic as well to match the client's signature?

Reply:
good catch, I guess we did not have tests for SetPerson on rollbar level